prefetch.bs

```diff
@@ -204,6 +245,16 @@ Given this, the non-prefetch case becomes:
     1. Run [=process a navigate response=] with |navigationType|, |allowedToDownload|, |hasTransientActivation|, and |navigationParams|.
 </div>
 
+Insert the following steps below the step that sets the request's [=request/destination=]:
+
+<div algorithm="process a navigate fetch patch">
```

Comment:
Can you just insert this directly into the algorithm above, since you rewrote it already? Maybe with a NOTE calling out that this step is new?

Reply:
Done.